rosdep/python.yaml

```diff
@@ -5998,6 +5998,10 @@ python3-pykdl:
     '*': [python3-pykdl]
     'bionic': null
     'xenial': null
+python3-pymodbus:
+  debian: [python3-pymodbus]
```

Comment:
This key isn't available in `stretch`.  It *is* available in `stretch-backports`, but we don't usually allow keys in the backports repository, as I understand it.  So I think this will have to get split out to a `null` key for `stretch` and `python3-pymodbus` for everything else.

Reply:
I didnt know stretch was still a required platform for new keys as it went EOL last month.
Addressed in https://github.com/ros/rosdistro/pull/26056/commits/85aee79a8a6de248b97f0596cfc9a2324b4d1025

Reply:
> I didnt know stretch was still a required platform for new keys as it went EOL last month.

I didn't realize that.  Thanks for the heads up; for now, I think we can leave the key since we've added it.